src/scripts/overview/view-in-jira-button.ts

```diff
@@ -0,0 +1,147 @@
+import { createJiraIcon } from '../icons/jira';
+
+export const addViewInJiraButton = (
+  JIRA_URL: string | null,
+  JIRA_PREFIX: string | null,
+) => {
+  if (!JIRA_URL || !JIRA_PREFIX) {
+    console.info(
+      "[Gitlab Extended] No Jira URL or prefix configured. If you want to be able to see the MRs issue in Jira, please configure the __gitlab-extended-jira_url and __gitlab-extended-jira_prefix in your local storage.",
```

Comment:
From a UX standpoint it'd be nice if it presets these to either empty strings or some value that can be used to determine it isn't set yet 

Having to copy and paste these keys in is a bit annoying